tests/provisioning-manifest-build.yml

```diff
@@ -4,7 +4,7 @@
   username: ${env:NEXUS_USERNAME}
   password: ${env:NEXUS_PASSWORD}
 - installBundle:
-  - 'mvn:org.jahia.modules/qa-module/3.1.0-SNAPSHOT'
+  - url: 'mvn:org.jahia.modules/qa-module/3.1.0-SNAPSHOT'
```

Comment:
Is `url` needed here? I'm looking at provisioning doc [here](https://github.com/Jahia/jahia-private/tree/master/bundles/provisioning) and not seeing if that's possible or not. 

If needed, maybe we should do the same for snapshot provisioning as well.

Reply:
It's to be inline with the one generated automatically